src/js/models/layerstructure.js

```diff
@@ -609,6 +609,20 @@ define(function (require, exports, module) {
     LayerStructure.prototype.byID = function (id) {
         return this.layers.get(id, null);
     };
+    
+    /**
+     * Get a list of Layer models by layer IDs.
+     *
+     * @param {Immutable.Iterable.<number>} ids
+     * @return {Immutable.Iterable.<Layer>}
+     */
+    LayerStructure.prototype.byIDs = function (ids) {
+        return ids.reduce(function (layers, id) {
```

Comment:
I also really don't like putting something like this so deep in the model that filters out null layers. Non-existent layer IDs are usually a sign of a bug.